CMakeLists.txt

```diff
@@ -54,6 +54,10 @@ else()
   message("Doing a local build")
 endif()
 
+if($ENV{NUPIC_ARM_BUILD})
```

Comment:
instead of setting the env variable manually, can you detect the ARM machine from cmake? Like Linux/Mac/Windows is set?